src/components/ProfileEditPOWnamePage.js

```diff
@@ -0,0 +1,124 @@
+import React from "react"
+import { useAuthActions, useAuthState } from "../auth"
+import { navigate } from "gatsby"
+import {
+  AppBar,
+  Button,
+  Toolbar,
+  IconButton,
+  TextField,
+  Paper,
+  Typography,
+  makeStyles,
+} from "@material-ui/core"
+import ArrowBackIcon from "@material-ui/icons/ArrowBack"
+
+// Is this CSS?
```

Comment:
This is css in javascript yes. So javascript syntax, but it is turned into css when the code is compiled.

Reply:
👍